pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/crd/kustomization.go

```diff
@@ -88,9 +88,13 @@ func (f *Kustomization) GetCodeFragments() machinery.CodeFragmentsMap {
 	if f.MultiGroup {
 		suffix = f.Resource.Group + "_" + f.Resource.Plural
 	}
-	// Generate resource code fragments
-	webhookPatch := make([]string, 0)
-	webhookPatch = append(webhookPatch, fmt.Sprintf(webhookPatchCodeFragment, suffix))
+
+	if !f.Resource.Webhooks.IsEmpty() {
+		webhookPatch := make([]string, 0)
+		// Change the line to uncomment the patch line
+		webhookPatch = append(webhookPatch, fmt.Sprintf(webhookPatchCodeFragment, suffix))
+		fragments[machinery.NewMarkerFor(f.Path, webhookPatchMarker)] = webhookPatch
+	}
```

Comment:
nit: does it make sense if we simplify the code lines like:
```suggestion
	if !f.Resource.Webhooks.IsEmpty() {
		webhookPatch := fmt.Sprintf(webhookPatchCodeFragment, suffix)
		fragments[machinery.NewMarkerFor(f.Path, webhookPatchMarker)] = []string{webhookPatch}
	}
```